modules/con_enabling-config_management_plugins_in_an_argo_cd_cr.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-config_management_plugins_in_an_argo_cd_cr_{context}"]
+= Enabling Config Management Plugins in an Argo CD CR
+
+Argo CD provides support for Helm, Jsonnet, and Kustomize as built-in config management tools. To use a different config management tool, or to enable features not provided by the built-in config management tools, you can use the Config Management Plugin (CMP).
+
+In the {gitops-title} Operator, the Config Management Plugin (CMP) is specified as a sidecar container for the Argo CD repo server container. For more information, see Config Management Plugins.
+
+In the {gitops-title} Operator, you can configure the Config Management plugin as a sidecar container in the Argo CD custom resource (CR). When you configure the sidecar container, you either specify an off-the-shelf or a custom-built container image. If you do not specify an image, the system uses the same image as the repo server for the plugin.
+
+To configure a sidecar container in the {gitops-title} Operator, add the `.spec.repo.sidecarContainers` key in the Argo CD CR. 
+
+.Example for `Config Management Plugin`
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+ name: my-plugin # <1>
```

Comment:
I'd change this to something like `name: my-argocd` since in general there's more to the argocd cr than just the plugin.

Reply:
Thanks for the feedback @jopit, I have changed the name of the Argo CD instance. Fixed.